ChangeLog.md

```diff
@@ -2357,116 +2357,116 @@
-  + remote: do not fail operations because of a migration
-  + remote: enable reloads for select remote connection options
-  + resources: allow for top output with or without percent sign in HealthCPU
-  + resources: Prevent an error message on stopping "Dummy" resource
-  + systemd: Prevent segfault when logging failed operations
-  + systemd: Reconnect to System DBus if the connection is closed
-  + systemd: set systemd resources' timeout values higher than systemd's own default
-  + tools: Do not send command lines to syslog
-  + tools: update SNMP MIB
-  + upstart: Ensure pending structs are correctly unreferenced
+  + **cib:** Do not terminate due to badly behaving clients
+  + **cman:** handle corosync-invented node names of the form Node{id} for peers not in its node list
+  + **controld:** replace bashism
+  + **crm_node:** Display node state with -l and quorum status with -q, if available
+  + **crmd:** resources would sometimes be restarted when only non-unique parameters changed
+  + **crmd:** fence remote node after connection failure only once
+  + **crmd:** handle resources named the same as cluster nodes
+  + **crmd:** Pre-emptively fail in-flight actions when lrmd connections fail
+  + **crmd:** Record actions in the CIB as failed if we cannot execute them
+  + **crm_report:** Enable password sanitizing by default
+  + **crm_report:** Allow log file discovery to be disabled
+  + **crm_resource:** Allow the resource configuration to be modified for `--force-{check,start,..}` calls
+  + **crm_resource:** Compensate for -C and -p being called with the child resource for clones
+  + **crm_resource:** Correctly clean up all children for anonymous cloned groups
+  + **crm_resource:** Correctly clean up failcounts for inactive anonymous clones
+  + **crm_resource:** Correctly observe `--force` when deleting and updating attributes
+  + **crm_shadow:** Fix "crm_shadow --diff"
+  + **crm_simulate:** Prevent segfault on arches with 64bit `time_t`
+  + **fencing:** ensure "required"/"automatic" only apply to "on" actions
+  + **fencing:** Return a provider for the internal fencing agent "#watchdog" instead of logging an error
+  + **fencing:** ignore stderr output of fence agents (often used for debug messages)
+  + **fencing:** fix issue where deleting a fence device attribute can delete the device
+  + **libcib:** potential user input overflow
+  + **libcluster:** overhaul peer cache management
+  + **log:** make syslog less noisy
+  + **log:** fix various misspellings in log messages
+  + **lrmd:** cancel currently pending STONITH op if stonithd connection is lost
+  + **lrmd:** Finalize all pending and recurring operations when cleaning up a resource
+  + **pengine:** Bug cl#5247 - Imply resources running on a container are stopped when the container is stopped
+  + **pengine:** cl#5235 - Prevent graph loops that can be introduced by "load_stopped -> `migrate_to"` ordering
```

Comment:
Either get rid of the double quotes or the ticks, and extend whichever is left to include the whole ordering.